publisher/product/ontology/build.sh

```diff
@@ -656,7 +670,7 @@ __HERE__
 
 	  ${FIND} . -mindepth 2 -name '*.ttl' -print | while read file; do quadify "$file"; done > "${DevQuadsFile}"
 	  echo "starting prod"
-	  ${GREP} -rl 'fibo-fnd-utl-av:hasMaturityLevel fibo-fnd-utl-av:Release' | \
+	  ${GREP} -rl '${ONTPUB_FAMILY}-fnd-utl-av:hasMaturityLevel ${ONTPUB_FAMILY}-fnd-utl-av:Release' | \
```

Comment:
The `-fnd-utl-av:hasMaturityLevel` and `-fnd-utl-av:Release` fragments are specific to FIBO.

Reply:
See above.